OpenOversight/app/static/js/browseFiltering.js

```diff
@@ -0,0 +1,70 @@
+// Set dropdown selections to what form passed in
+document.getElementById("race").value = "{{ race }}";
+document.getElementById("gender").value = "{{ gender }}";
+document.getElementById("rank").value = "{{ rank }}";
+document.getElementById("min_age").value = "{{ min_age }}";
+document.getElementById("max_age").value = "{{ max_age }}";
+
+{% if officers.next_num %}
```

Comment:
this Jinja2 templating shouldn't work in the JavaScript no? this will produce a `SyntaxError` (as is the pagination on this branch doesn't work for me, I think due to this issue)